util.py

```diff
@@ -0,0 +1,12 @@
+def convert_str_to_bool(value: str, default: bool) -> bool:
+    try:
+        if value is None:
+            return default
+        if value.lower() == 'true':
+            return True
+        elif value.lower() == 'false':
+            return False
+        else:
+            return default
+    except Exception as e:
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Too broad exception handling</b></div>
<div id="fix">

Catching a broad 'Exception' may hide bugs. Consider catching specific exceptions that you expect might occur.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
    except AttributeError:
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rkj6124/test/pull/2#issuecomment-2557465685>#6d7189</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged